common/script/fns/statsComputed.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+import content from '../content/index';
+import splitWhitespace from '../libs/splitWhitespace';
+const $w = splitWhitespace;
+import * as statHelpers from '../statHelpers';
+
+module.exports = function statsComputed (user) {
+  let thisArr = $w('stats stats.buffs items.gear.equipped.weapon items.gear.equipped.armor items.gear.equipped.head items.gear.equipped.shield');
+  let computed = _.reduce(['per', 'con', 'str', 'int'], (m, stat) => {
+    m[stat] = _.reduce(thisArr, (m2, path) => {
+      let val = user.fns.dotGet(path);
+      let item = content.gear.flat[val];
+      if (!item) item = {};
+      if (!item[stat])
```

Comment:
yeah it should be `if () { } else {}`. Otherwise linting will fail and if it doesn't I still don't like it :P